aw-server/src/android/mod.rs

```diff
@@ -165,9 +165,16 @@ pub mod android {
         _: JClass,
         java_dir: JString,
     ) {
-        let path = &jstring_to_string(&env, java_dir);
-        debug!("Setting android data dir as {}", path);
-        dirs::set_android_data_dir(path);
+        match jstring_to_string(&env, java_dir) {
+            Some(path) => {
+                debug!("Setting android data dir as {}", path);
+                dirs::set_android_data_dir(&path);
+            }
+            None => {
+                // Failed to convert string - do nothing and return
```

Comment:
Wouldn't it make sense to debug log? Error log or smth? 

What happens if you can't set the data dir? Does the rest handle that somewhat sanely?

Reply:
yeah, a debug log makes more sense, here. Will do.